packages/vscode-ui5-language-assistant/README.md

```diff
@@ -1,56 +1,54 @@
-# UI5 Language Support
+# SAPUI5 Language Support
```

Comment:
The approved SAP Branding name here was UI5 Language Assistant
I will rename support -> Assistant, but cannot add `SAP` prefix